en/sensor/pmw3901.md

```diff
@@ -0,0 +1,76 @@
+# PMW3901 Flow Sensor
+
+PMW3901 is an [optical flow](../sensor/optical_flow.md) asic that computes the flow internally and provides a difference in pixels between 
+each frame over a SPI interface. It is basically a tracking sensor similar to what you would find in a computer mouse, but adapted to work between 
```

Comment:
"an SPI" (FYI, the rule in English is that the "an" is used before word that _sounds_ like it has a vowel first. So "an esssPI").